src/main/java/core/basesyntax/FieldXY.java

```diff
@@ -13,7 +13,44 @@
 public class FieldXY {
 
     public void moveRobot(Robot robot, int toX, int toY) {
-        //your code
+        Direction directToX;
+        Direction directToY;
+        if (robot.getX() > toX) {
+            directToX = Direction.LEFT;
+            while (robot.getDirection() != directToX) {
+                robot.turnLeft();
+            }
+            while (robot.getX() != toX) {
+                robot.stepForward();
+            }
+        }
+        if ((robot.getX() < toX)) {
+            directToX = Direction.RIGHT;
+            while (robot.getDirection() != directToX) {
+                robot.turnLeft();
+            }
+            while (robot.getX() != toX) {
+                robot.stepForward();
+            }
```

Comment:
this code is duplicated in lines 23-25
try to simplify it

Reply:
done